pom.xml

```diff
@@ -90,21 +90,40 @@
 
     <dependencies>
         <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
-            <version>1.2.17</version>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.5.18</version>
+            <scope>compile</scope>
```

Comment:
I'm not sure to understand the scope here.
If it's compiled, this should not be present in the jar, so something else will provide it at runtime ?